otx/cli/tools/train.py

```diff
@@ -244,10 +245,12 @@ def main():  # pylint: disable=too-many-branches
         assert resultset.performance is not None
         print(resultset.performance)
 
-    task.cleanup()
-
     if args.gpus:
         multigpu_manager.finalize()
+    elif is_multigpu_child_process():
+        sys.exit()
```

Comment:
Is such a hard exit necessary here?
Logic like
```python
if not is_multigpu_child_process():
   task.cleanup()
```
looks more natural

Reply:
you're right. I'll update it.